scalameta/contrib/src/main/scala/scala/meta/contrib/DocTree.scala

```diff
@@ -1,11 +1,27 @@
 package scala.meta.contrib
 
+import org.scalameta.adt._
+
 import scala.util.matching.Regex
 
 /**
-  * Represents a scaladoc line.
+  * Represents a scaladoc element.
+  */
+@root
+trait DocTree {
```

Comment:
I don't think the `name/body` fields should be on `DocToken`. I had in mind that the ast would be more like here https://github.com/scalameta/scalameta/pull/692#pullrequestreview-24225248